.github/workflows/lint.yml

```diff
@@ -14,7 +14,10 @@ jobs:
   # From https://github.com/golangci/golangci-lint-action
   golangci:
     name: lint
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Should we add `windows-latest`?

Reply:
I think we should, yes.

We could also bump up the golangci-lint version to v1.47 https://github.com/golangci/golangci-lint/releases

Reply:
@francislavoie @mholt 
Hi, I'm trying to be upped the golangci-lint version, Although, I found the lint error(It has not occurred in the old version)
I consider that it should treat another issue once.

https://pkg.go.dev/github.com/google/cel-go/cel#Ast.ResultType
https://pkg.go.dev/github.com/google/cel-go/cel#Functions

```
run golangci-lint
  Running [/home/runner/golangci-lint-1.47.2-linux-amd64/golangci-lint run --out-format=github-actions] in [] ...
  Error: SA1019: checked.ResultType is deprecated: use OutputType  (staticcheck)
  Error: SA1019: checked.ResultType is deprecated: use OutputType  (staticcheck)
  Error: SA1019: cel.Functions is deprecated: use Function() instead to declare the function, its overload signatures, and the overload implementations.  (staticcheck)
  Error: SA1019: cel.Functions is deprecated: use Function() instead to declare the function, its overload signatures, and the overload implementations.  (staticcheck)
  Error: SA1019: cel.Functions is deprecated: use Function() instead to declare the function, its overload signatures, and the overload implementations.  (staticcheck)
```

Reply:
Solving that requires uncommenting the line:

```
# only-new-issues: true
```

I vote for uncommenting the line, otherwise the lint errors might confuse the PRs with multiple unrelated changes.

Reply:
I am actually working on a fix for those errors as I type this.

I got it to compile but now the tests fail related to those `Function()` changes... I'm not sure how to use it properly.